libs/cloud-database/src/postgres/database.rs

```diff
@@ -167,12 +170,14 @@ impl PostgreSQL {
     pub async fn create_user(&self, user: CreateUser) -> sqlx::Result<Option<(User, Workspace)>> {
         let mut trx = self.db.begin().await?;
         let create_user = "INSERT INTO users 
-            (name, password, email, avatar_url)
+            (uuid, name, password, email, avatar_url)
             VALUES ($1, $2, $3, $4)
         ON CONFLICT (email) DO NOTHING
-        RETURNING id, name, email, avatar_url, created_at";
+        RETURNING uuid as id, name, email, avatar_url, created_at";
 
+        let uuid = Uuid::new_v4().to_string();
```

Comment:
can we switch to nanoid to keep uuid shorter? https://docs.rs/nanoid/latest/nanoid/